pages/Plan/[id].js

```diff
@@ -1,16 +1,26 @@
 import { useRouter } from 'next/router';
```

Comment:
```suggestion
```

Reply:
Since you don't use the useRouter in this file, the import should be removed here.